pkg/volume/portworx/portworx_util.go

```diff
@@ -23,16 +23,18 @@ import (
 	volumeclient "github.com/libopenstorage/openstorage/api/client/volume"
 	osdspec "github.com/libopenstorage/openstorage/api/spec"
 	osdvolume "github.com/libopenstorage/openstorage/volume"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/kubernetes/pkg/api/v1"
 	"k8s.io/kubernetes/pkg/volume"
 )
 
 const (
-	osdMgmtPort      = "9001"
-	osdDriverVersion = "v1"
-	pxdDriverName    = "pxd"
-	pwxSockName      = "pwx"
-	pvcClaimLabel    = "pvc"
+	osdMgmtPort         = "9001"
+	osdDriverVersion    = "v1"
+	pxdDriverName       = "pxd"
+	pwxSockName         = "pwx"
+	pvcClaimLabel       = "pvc"
+	labelNodeRoleMaster = "node-role.kubernetes.io/master"
```

Comment:
In latest incremental (4164374), I decided to remove filtering based on a specific label. 

Previously the goal was to filter out master nodes but for that we needed to label master nodes during deployement. We weren't gaining enough for this added step so I decided to remove the filtering altogether.